core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/impl/api/parse.kt

```diff
@@ -388,6 +388,8 @@ internal object Parsers : GlobalParserOptions {
                 null
             }
         },
+        // Char
+        stringParser<Char> { it.singleOrNull() },
```

Comment:
not necessarily, columns with single characters can be quite common, especially when reading pivoted data, plus it could save memory, which is always a good thing, right? :)